### DISCUSSION TITLE
@adonisjs/lucid/providers/LucidProvider module not found

### DISCUSSION THREAD

**Discussion by danielperezp** [adonisjs/core#4352] [Help]
Hi all, I'm new to the @adonisjs/lucid configuration, I have problems updating the version of the @adonisjs/lucid package from 6.2.1 to 18.0.0

I'm using docker image node:16.20.1-alpine to containerize the app

I have this folder structure and providers imports approach

 
![Screenshot 2024-01-24 at 12 39 39 PM](https://github.com/adonisjs/core/assets/13216958/4527bf4c-a920-4ac8-a72e-c9e5d7a3e77e)

I reviewed the latest docs of lucid here https://lucid.adonisjs.com/docs/installation
but I'm not using typescript.

This is the issue when the docker container tries to run the app
<img width="565" alt="Screenshot 2024-01-24 at 12 45 08 PM" src="https://github.com/adonisjs/core/assets/13216958/2c4765ea-c150-4500-ae1a-569340101d05">

I checked if the @adonisjs/lucid/providers/LucidProvider is downloaded in node_modules and there is that file
![Screenshot 2024-01-24 at 12 48 23 PM](https://github.com/adonisjs/core/assets/13216958/ada6171a-a2fe-4fad-8c7e-34a184aa0f15)

when I use the lucid version 6.2.1 the app works but when I update it to 18.0.0 it doesn't.

Should I change the way that I'm importing the providers?

Thanks in advance.

**Comment by RomainLanz** [comment]
Hey @danielperezp! 👋🏻 

It seems you are still using AdonisJS 4 (or even AdonisJS 3). The latest version of Lucid is for AdonisJS 6.

**Reply by danielperezp** [reply]
Hey @RomainLanz, thank you, so what is the latest compatible version of lucid to continue working with adonisJS 4?
